<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PredictiveText</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>461</width>
     <height>267</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QHBoxLayout" name="entryLayout">
      <item>
       <layout class="QVBoxLayout" name="labelGroup">
        <item>
         <widget class="QLabel" name="begWordLabel">
          <property name="text">
           <string>Beginning of Word:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="predWordLabel">
          <property name="text">
           <string>Predicted Word:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="wordLayout">
        <item>
         <widget class="QLineEdit" name="wordEntry">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="predictionLayout">
          <item>
           <widget class="QLabel" name="predictedWord">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="useWordButton">
            <property name="text">
             <string>Use Word</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="treeButtonLayout">
        <item>
         <widget class="QPushButton" name="saveTreeButton">
          <property name="text">
           <string>Save Tree</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reloadTreeButton">
          <property name="text">
           <string>Reload Tree</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="separator">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="textLayout">
      <item>
       <widget class="QPlainTextEdit" name="enteredText">
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="textButtonLayout">
        <item>
         <widget class="QPushButton" name="saveTextButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Save Text</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearTextButton">
          <property name="text">
           <string>Clear Text</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
